setup.py

```diff
@@ -32,7 +32,7 @@
 ]
 test_deps = [
     'modernize',
-    'nose>=1.3.7',
+    'nose @ git+https://github.com/dimagi/nose.git@06dff28bbe661b9d032ce839ea0ec8e9eaf6f337',
```

Comment:
Looks like `nose` hasn't received an update since 2015. Is it time to remove it from our test stack here? (Not blocking this PR.)

Reply:
This is a bigger topic, so I agree that it should not block this PR. My vote is to keep nose until pytest has a solution for https://github.com/pytest-dev/pytest/issues/3834